src/lib/routing/router.ts

```diff
@@ -20,21 +20,47 @@ const defaultErrorHandler = (auto405) => async (err: Error, req: EReq, res: ERes
   res.withStatus(500).json({ error: err.message });
 }
 
+const preflightHandler = (autoCorsPreflight: AutoCorsPreflightOptions) => async (req: EReq, res: ERes) => {
+  if(autoCorsPreflight.trustedOrigins.length===0){
+    return;
+  }
+  let originHeaderValue :string | null = null;
+  if(autoCorsPreflight.trustedOrigins.length===1 && autoCorsPreflight.trustedOrigins[0] === '*'){
+    originHeaderValue = '*';
+  }else{
+    const origin = req.headers.get("origin");
+    if (autoCorsPreflight.trustedOrigins.includes(origin)){
+      originHeaderValue = origin;
+    }
+  }
+  if(originHeaderValue==null){
+    return;
```

Comment:
This `return` (and also the one above) happen when we want to deny the CORS preflight, right?  But does returning here do that?  What's the outcome for the response to the client?  It ought to be a 403 I think (forbidden)